lua/better_escape.lua

```diff
@@ -49,73 +51,74 @@ local function clear_mappings()
 end
 
 -- WIP: move this into recorder.lua ?
-local last_key = nil
+local recorded_key = nil -- When a startkey is pressed, `recorded_key` is set to it (e.g. if jk is a mapping, when 'j' is pressed, `recorded_key` is set to 'j')
```

Comment:
nit: should move this to the above line and perhaps split it up so it's no too long